modules/serverless-kafka-tls.adoc

```diff
@@ -0,0 +1,80 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+== Configuring TLS authentication
```

Comment:
Same comment as above. Module titles should always be `= Title of module`, vs `== Title of module`. You can adjust the heading level in the include statement in the assembly. 👍